ui/js/models/perf/alerts.js

```diff
@@ -117,7 +118,7 @@ treeherder.factory('PhAlerts', [
 
             const formatAlert = function (alert, alertList) {
                 return _.padStart(alert.amount_pct.toFixed(0), 3) + '%  ' +
-                (alert.title.padEnd(_.max(alertList, function (alert) { return alert.title.length; }).title.length + 5)) +
```

Comment:
The existing `_.max()` usage here seems broken. As far as I can tell, the function only accepts one argument:
https://lodash.com/docs/#max
https://github.com/lodash/lodash/blob/4.17.10/lodash.js#L16273-L16277

Reply:
would it be better to explicitly define a `getAlertWithMaximumLength()` function with local scope? This might add to the readability:

```javascript
            const getMaximumAlertLength = alertList => Math.max(...alertList.map(alert => alert.title.length));

            const formatAlert = function (alert, alertList) {
                return _.padStart(alert.amount_pct.toFixed(0), 3) + '%  ' +
                (alert.title.padEnd(getMaximumAlertLength(alertList) + 5)) +
                displayNumberFilter(alert.prev_value) + ' -> ' + displayNumberFilter(alert.new_value);
```

Reply:
@SirR4T : Yeah, that looks like the best approach.  This was clearly not doing the right thing before, so it looks like no one noticed or cared.  :)  But your approach looks like a good fix.  Thanks!

Reply:
thanks @camd , pushed now.